courses/models_test.py

```diff
@@ -148,6 +148,33 @@ def test_course_run_past(end_days, expected):
     assert CourseRunFactory.create(end_date=end_date).is_past is expected
 
 
+@pytest.mark.parametrize(
+    "start_delta, end_delta, expiration_delta, exceptionExpected",
```

Comment:
rename `exceptionExpected`  to `exception_expected `

Reply:
Updated